src/components/CampaignOverview.vue

```diff
@@ -0,0 +1,167 @@
+<template>
+  <section class="favor-ranking">
+    <table class="favor-ranking-table tableFixHead">
```

Comment:
According to sonar rules `<table>` tags should have a description:
https://rules.sonarsource.com/html/RSPEC-1085

Btw, you can install SonarLint ext for VSCode from here: https://marketplace.visualstudio.com/items?itemName=SonarSource.sonarlint-vscode